runner/app/live/trickle/trickle_subscriber.py

```diff
@@ -56,8 +56,8 @@ async def preconnect(self):
                 resp.release()
                 logging.error(f"Trickle sub Failed GET {url} status code: {resp.status}, msg: {body}")
 
-            except aiohttp.ClientError as e:
-                logging.error(f"Trickle sub Failed to complete GET {url} error: {e}")
+            except Exception:
+                logging.exception(f"Trickle sub Failed to complete GET {url}", stack_info=True)
```

Comment:
Wow I thought `stack_info` was precluded with `logging.exception`. Apparently not from func signature at least 🧐